README.rst

```diff
@@ -53,7 +53,7 @@ And to get the **latest development version**, you can use:
 
 .. code-block::
 
-    $ pip install -U https://github.com/benchopt/benchopt/archive/master.zip
+    $ pip install -U git+https://github.com/benchopt/benchopt.git#egg=benchopt
```

Comment:
wait does it mean now one need git to install this? 

I would not enforce to have git for this

```suggestion
    $ https://github.com/benchopt/benchopt/archive/master.zip
```

Reply:
edited above...

Reply:
from the comment in the PR description, I think @mathurinm said replacing `master` by `main` does not work.

Reply:
```suggestion
    $ pip install -i https://test.pypi.org/simple/ benchopt
```

Reply:
Taking a step back here: do we actually need this ? If I need to install the truly bleeding edge version, the standard way is to clone the repo and run `pip install -e .`, no ?

Reply:
This comment is here because we are not super good on release and backward compat yet.
So often, once need the master branch version to run a benchmark. I would put the install from test.pypi as this seems an easy fix.
We can remove this once we improve on the compat/release.

WDYT?